datamodel/low/v3/paths.go

```diff
@@ -152,12 +175,11 @@ func (p *Paths) Build(ctx context.Context, _, root *yaml.Node, idx *index.SpecIn
 			path := new(PathItem)
```

Comment:
Here is the problem. 

Not sure why, but the parent context is being passed down again.  In the `TranslatePipeline` call, we need to make sure we pull the context when looking up a ref node, and pass it down. Change the code to look like this (sorry I can' t make suggestions on code that has not changed)
<img width="816" alt="Screenshot 2023-12-04 at 12 16 46 PM" src="https://github.com/pb33f/libopenapi/assets/187345/5ebd8915-2f12-4b70-9c05-7939ea51d490">

Reply:
the test passes once this change is made.